soc/arm/atmel_sam0/common/Kconfig.saml2x

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) 2021 Argentum Systems Ltd.
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_SERIES_SAML21
+
+config SOC_ATMEL_SAML_DEBUG_PAUSE
+	bool "Insert a pause at boot, to allow a debugger to attach"
+	default y
+	help
+		Issues have been observed while attempting to attach a debugger.
+		These can be mitigated by inserting a small delay during the early boot
+		sequence, before the system clock is configured.
+
+		If you ever intend to attach a debugger, say y.
+		If you are confident that you will never attach a debugger to the
+		resulting binary, or require the best possible boot time, say n.
```

Comment:
The formatting is wrong here. Please update this to use the following style:

```
help
<TAB><SPACE><SPACE>Issues have been observed...
````

Instead of

```
help
<TAB><TAB>Issues have been observed
```

Similar comments apply elsewhere.

I know it's weird, but it's how Linux does it, so we do the same.